src/model/request/TransferAdaBlockchain.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  IsNotEmpty,
+  IsNumberString,
+  IsOptional,
+  IsString,
+  Length,
+} from 'class-validator'
+import { TransferBtcBasedBlockchain } from './'
+
+export class TransferAdaBlockchain extends TransferBtcBasedBlockchain {
+  @IsNumberString()
+  @IsOptional()
+  public fee: string
+
+  @IsNotEmpty()
+  @IsString()
+  @Length(30, 110)
+  public changeAddress: string
```

Comment:
why didnt you update the transferADA logic here already? it wont work for tatumJS users after this.

Reply:
Hmm, okay. I updated logic in core-api repo. I did not notice that logic is in tatum-js as well. sorry for that. I gonna add it there.

Reply:
I added corresponding changes to tatum-js and retested.